src/Transactions/TransactionDetails/TransactionInformation/utils.js

```diff
@@ -0,0 +1,10 @@
+import { TRANSACTION_SOURCE } from '../../constants';
+
+export function getSourceLink(source, fiscalYearId = '', invoiceId = '', invoiceLineId = '', poLineId = '') {
```

Comment:
here as well, I see no need in this atomic attributes. IMHO, it's OK to have function `getSourceLink` that accepts in itself `transaction` object and returns link to its source